src/validations/validateExpireDate.ts

```diff
@@ -23,12 +17,12 @@ export const validateExpireDate = (
     };
   }
 
-  if (key === 'year' && parseInt(value) < parseInt(currentYear)) {
+  if (key === 'year' && parseInt(value) < currentYear) {
```

Comment:
변수화해볼 수 있을거같아요. 더 읽기쉽게!

```suggestion
  if (key === 'year') {
    const isPastYear = parseInt(value) < currentYear;
    if (isPastYear) {

    }

```

Reply:
수정했습니다!

fa93264b35bf1665db8d13835b3fb784a3e9547a